### DISCUSSION TITLE
TC  doesn't work with QR Codes for GO-PRO 11 after latest camera update

### DISCUSSION THREAD

**Discussion by rconnew** [gopro/labs#1081] [General]
After my GO-PRO11 was updated to the latest update the labs function of reading QR codes no longer works, in fact it sends the camera into a cycle of reboots every time I try to set the TC with a QR code. Tried uninstalling and reinstalling but still the same issue. Anyone know how to fix this please?

**Comment by dnewman-gpsw** [comment]
Firmware version?
QR Code used?
Factory Reset?

This is not a known fault.

**Reply by rconnew** [reply]
Same problem on both my GO-PRO 11's, so not down to an individual camera issue.
Factory reset both and both have the same issue

GO RPO 11 - FIRMWARE UPDATE Download Update v02.01.00

LABS_HERO11_02_10_70


Using Tentacle Timebar 0.13.0 (30) to produce the QR code for the GO-PRO 11 to look at.

To rule out Tentacle Timebar I also tried with the time page here https://gopro.github.io/labs/control/precisiontime/

GP-PRO 11 reads the code and the tick box appears then it goes odd and starts rebooting. You have to take the battery out to stop it.

**Comment by dnewman-gpsw** [comment]
That is old firmware. Install the latest v2.30.70, March 2024

Current firmware is always here: https://gopro.github.io/labs/

P.S. Still have never heard of this fault. Have there cameras had other modifications? This is not an issue with any Labs firmware or camera model.

**Reply by rconnew** [reply]
That was my mistake I copied the wrong file name. It has the correct 2.30.70 installed.
See a video of the issue here https://vimeo.com/1011641132?share=copy#t=0
Cameras are both standard nothing different about them at all.

**Reply by dnewman-gpsw** [reply]
That is extremely weird. No clue, not on any camera here.  I need to learn what is different, or you will have to use stock firmware.

**Reply by rconnew** [reply]
I tried an older firmware 2.2 with Labs 02 03 70 but that doesn't fix it. Where can I find an older version of Labs for GO PRO 11 please?

**Reply by dnewman-gpsw** [reply]
It is not the firmware as the current works fine on my camera and there have been on other reports.  There is something else involved.

**Reply by rconnew** [reply]
Would you please send me a link to the later one then I can see as that one did work in October 2023 which was the last time I used it. Thanks

**Reply by dnewman-gpsw** [reply]
I don't believe you will find it to be firmware.  All published firmware is in this GitHub repo. https://github.com/gopro/labs/tree/master/docs/firmware/lfs

**Reply by rconnew** [reply]
02_20_70 works  with GO PRO 11 -  2.20 firmware

**Reply by rconnew** [reply]
OK I think I have the problem. The phone app updated my GP11 to V2.30 and its when you update to LABS 02 30 70 from that update the issue happens. If you update from LABS 02_20_70 with GP 11 - 2.20 firmware installed initially and don't upgrade to V2.30 then it all works fine. So my guess is that is where the issue is.
